### ISSUE TITLE
Discussão sobre escopo dos temas do repositório.

### ISSUE THREAD

**Issue by Bluniz** [he4rt/react4noobs#78]
Bom galera, gostaria de discutir com todos sobre a grade de conteudos.

Gostaria de saber o que vocês sentem falta no repositório, oque pode melhorar e oque não é necessário.

Alguns pontos que pretendo fazer:
- Tópico sobre Renderização condicional
- Lista e Chaves
// Esses 2 itens são literalmente o que falta para fechar a introdução a React.

Algumas coisas que não vejo necessidade visto a natureza do projeto(voltado a iniciantes)

5. NextJS
6. Gatsby

Por que não vejo necessidade? Pelo simples fato de ambos serem Frameworks baseadas em React, ambas tem um mundo de funcionalidades que necessitam de uma abordagem mais descritiva e com um passo a passo, no estilo deste repositório.

Como sugestão: Um futuro next4noobs e gatsby4noobs (Se já não existir), com pré conhecimento necessário, Javascript e React.

**Comment by Iam7K** [comment]
Boa noite, perdoe pela demora. 

Acabei de ler e tenho algo a pontuar:

• Não é necessário remover os tópicos já criados, não por agora. Até que haja uma nova proposta para execução dos tópicos citados separando as frameworks.

A proposta do 4noobs é que qualquer um pode contribuir com os conteúdos da comunidade, portanto deve-se manter até que algum voluntário se disponha a desenvolver.

**Comment by Rychillie** [comment]
> Bom galera, gostaria de discutir com todos sobre a grade de conteudos.
> 
> 
> 
> Gostaria de saber o que vocês sentem falta no repositório, o que pode melhorar e o que não é necessário.
> 
> 
> 
> Alguns pontos que pretendo fazer:
> 
> - Tópico sobre Renderização condicional
> 
> - Lista e Chaves
> 
> * Esses 2 itens são literalmente o que falta para fechar a introdução a React.
> 
> 
> 
> Algumas coisas que não vejo necessidade visto a natureza do projeto(voltado a iniciantes)
> 
> 
> 
> 5. NextJS
> 
> 6. Gatsby
> 
> 
> 
> Por que não vejo necessidade? Pelo simples fato de ambos serem Frameworks baseadas em React, ambas tem um mundo de funcionalidades que necessitam de uma abordagem mais descritiva e com um passo a passo, no estilo deste repositório.
> 
> 
> 
> Como sugestão: Um futuro next4noobs e gatsby4noobs (Se já não existir), com pré conhecimento necessário, Javascript e React.

Concordo que não necessita a implementação de NextJS + Gatsby e gostaria de complementar adicionando importância de trabalharmos Funções, Hooks e Renderização

**Comment by vensauro** [comment]
@Rychillie pode me explicar o que são esses tópicos que você acha importante? não entendi muito, e gostaria de ajudar escrevendo sobre


E realmente acham que não pode ter nem um tópico geral sobre o nextjs/gatsby o qual linka para o futuro 4noobs se tiver?
Digo isso pois eu estava começando a escrever sobre gatsby e tem um PR aberto sobre nextjs que parece faltar pouca coisa (que talvez dê para refatorar para outro rep)

**Comment by vensauro** [comment]
Apesar que no geral concordo que é algo grande, eu tentei fazer uma lista simples de coisas importantes.
A qual ainda falta informação para ser abordada (apesar de ser algo avançado e desnecessário, preciso apenas para quem vai fazer algo super avançado)

- why use gatsby
- what's the diff between gatsby and cra/vite
- basic web rendering showing csr and ssg
- how to start using
    - templates
- how to make pages and components
- routing
- how to get data using graphql
- how config plugins and get more data
- gatsby image
- styling
- create page programating
